src/clients/channel-access-token/test/Model/IssueChannelAccessTokenResponseTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/**
+ * IssueChannelAccessTokenResponseTest
+ *
+ * PHP version 7.4
+ *
+ * @category Class
+ * @package  LINE\Clients\ChannelAccessToken
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ */
+
+/**
+ * Channel Access Token API
+ *
+ * This document describes Channel Access Token API.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Generated by: https://openapi-generator.tech
+ * OpenAPI Generator version: 6.5.0
+ */
+
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Please update the test case below to test the model.
+ */
+
+namespace LINE\Clients\ChannelAccessToken\Test\Model;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * IssueChannelAccessTokenResponseTest Class Doc Comment
+ *
+ * @category    Class
+ * @description Issued channel access token
+ * @package     LINE\Clients\ChannelAccessToken
+ * @author      OpenAPI Generator team
+ * @link        https://openapi-generator.tech
+ */
+class IssueChannelAccessTokenResponseTest extends TestCase
+{
+
+    /**
+     * Setup before running any test case
+     */
+    public static function setUpBeforeClass(): void
+    {
+    }
+
+    /**
+     * Setup before running each test case
+     */
+    public function setUp(): void
+    {
+    }
+
+    /**
+     * Clean up after running each test case
+     */
+    public function tearDown(): void
+    {
+    }
+
+    /**
+     * Clean up after running all test cases
+     */
+    public static function tearDownAfterClass(): void
+    {
+    }
+
+    /**
+     * Test "IssueChannelAccessTokenResponse"
+     */
+    public function testIssueChannelAccessTokenResponse()
+    {
+        // TODO: implement
+        $this->markTestIncomplete('Not implemented');
+    }
+
+    /**
+     * Test attribute "accessToken"
+     */
+    public function testPropertyAccessToken()
+    {
+        // TODO: implement
+        $this->markTestIncomplete('Not implemented');
```

Comment:
nits: If there's no plan to implement here, you can skip the test code generation.

Reply:
Thanks. ignore tests using `.openapi-generator-ignore`.